webhook_handlers/tests/test_gitlab.py

```diff
@@ -303,317 +293,19 @@ def test_handle_system_hook_project_create(self):
                 "project_visibility": "private",
             },
         )
-        assert response.status_code == status.HTTP_200_OK
-        assert response.data == "Repository created"
+        assert response.status_code == status.HTTP_403_FORBIDDEN
 
-        new_repo = Repository.objects.get(
+        new_repo = Repository.objects.filter(
             author__ownerid=owner.ownerid, service_id=project_id
-        )
-        assert new_repo is not None
-        assert new_repo.private is True
-        assert new_repo.name == "storecloud"
-
-    def test_handle_system_hook_project_destroy(self):
```

Comment:
are these all deleted because we don't intend to support these system hooks without an enterprise license? should we assert that a PermissionDenied() is raised?

Reply:
yes - System hooks are only for IS_ENTERPRISE, and it does a PermissionDenied() is raised test in `test_handle_system_hook_not_enterprise`.
I deleted them because it seemed redundant to test every type of system hook - but your comment made me realize that I should add them back to make it clear that this is the intended behavior.